server/database/index.js

```diff
@@ -5,20 +5,6 @@ var connection = mysql.createConnection({
   password: 'Alex',
   database: 'review_data'
 });
+
 connection.connect();
 
-module.exports.accessor = function(queryString, queryArgs) {
-  return new Promise((resolve, reject) => {
-    connection.query(queryString, queryArgs, (err, res) => {
```

Comment:
I think it looks really good and you fixed that issue with the callback and promise set up.